client/extensions/woocommerce/app/dashboard/setup-header.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+const SetupHeader = ( { imageSource, imageWidth, subtitle, title } ) => {
+	return (
+		<div className="dashboard__setup-header">
+			{ imageSource ? <img src={ imageSource } width={ imageWidth } className="dashboard__setup-header-image" /> : null }
+			{ <h2 className="dashboard__setup-header-title">{ title }</h2> }
+			{ subtitle ? <h3 className="dashboard__setup-header-subtitle">{ subtitle }</h3> : null }
```

Comment:
If we use the syntax I've come to favor so much, we could drop the ternary and `null` in markup for checks like this:

`{ subtitle && ( <h3 className="dashboard__setup-header-subtitle">{ subtitle }</h3> ) }`